src/services/createRouterReject.ts

```diff
@@ -46,18 +46,17 @@ export function createRouterReject({
       state: {},
     }
 
-    const resolved = {
+    return {
       id: route.id,
       matched: route,
       matches: [route],
       name: type,
       query: createResolvedRouteQuery(''),
       params: {},
       state: {},
+      href: '/',
```

Comment:
Hmm hard coding this to '/' is interesting. This is really only used for NotFound when the initialUrl doesn't match anything right?

Reply:
yes that's where we use this reject route. The actual `href` of the rejection route is probably never actually used anywhere but required to satisfy the type